reference/OpenAPI.json

```diff
@@ -24023,14 +24037,105 @@
+        "description": "Response body schema for **PUT** `/v1/loyalties/{campaignId}/rewards/{assignmentId}`.",
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/RewardsAssignmentsUpdateRequestBody"
+          }
+        ]
+      },
+      "LoyaltiesRewardsUpdateAssignmentResponseBody": {
+        "type": "object",
+        "title": "Loyalties Rewards Update Assignment Response Body",
+        "description": "Response body schema for **PUT** `/v1/loyalties/{campaignId}/rewards/{assignmentId}`.",
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/RewardAssignment"
+          }
+        ]
+      },
+      "LoyaltiesRewardsCreateAssignmentResponseBody": {
+        "type": "object",
+        "title": "Loyalties Rewards Create Assignment Response Body",
+        "description": "Response body schema for **POST** `/v1/loyalties/{campaignId}/rewards`.",
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/RewardAssignment"
+          }
+        ]
+      },
+      "LoyaltiesRewardsCreateAssignmentRequestBody": {
+        "title": "Loyalties Rewards Create Assignment Request Body",
         "type": "array",
-        "description": "Response body schema for **POST** `/loyalties/{campaignId}/rewards`.",
+        "description": "Request body schema for **POST** `/loyalties/{campaignId}/rewards`.",
         "items": {
-          "$ref": "#/components/schemas/4_obj_reward_assignment_object"
+          "$ref": "#/components/schemas/LoyaltiesRewardsCreateAssignmentItemRequestBody"
         }
       },
+      "LoyaltiesRewardsCreateAssignmentItemRequestBody": {
+        "title": "Loyalties Rewards Create Assignment Item Request Body",
+        "oneOf": [
+          {
+            "$ref": "#/components/schemas/LoyaltiesRewardsCreateAssignmentItemDigitalOrMaterialRequestBody"
+          },
+          {
+            "$ref": "#/components/schemas/LoyaltiesRewardsCreateAssignmentItemCoinRequestBody"
+          }
+        ]
+      },
+      "LoyaltiesRewardsCreateAssignmentItemDigitalOrMaterialRequestBody": {
+        "title": "Loyalties Rewards Create Assignment Item Digital Or Material Request Body",
+        "properties": {
+          "reward": {
+            "type": "string",
+            "description": "The reward ID of the campaign to which the campaign is to be assigned."
+          },
+          "parameters": {
+            "type": "object",
+            "description": "Defines the cost of the reward.",
+            "required": [
+              "loyalty"
+            ],
+            "properties": {
+              "loyalty": {
+                "type": "object",
+                "description": "Defines the equivalent points value of the reward.",
+                "properties": {
+                  "points": {
+                    "type": "integer",
+                    "description": "Number of points that will be subtracted from the loyalty card points balance if the reward is redeemed."
```

Comment:
Dodaj zdanie, kinda obvious, ale niech mają: `Must be a positive integer`.

Można dodać też `"minimum": 1`

Analogicznie w schemacie `RewardsAssignmentsCreateDigitalOrMaterialRewardRequestBody`.